### DISCUSSION TITLE
maybe i find a bug when tranning assist2017

### DISCUSSION THREAD

**Discussion by wulongxiaomeem** [pykt-team/pykt-toolkit#51] [Q&A] [answered]
CUDA_VISIBLE_DEVICES=2 nohup python wandb_dkt_train.py --dataset_name=assist2017 --use_wandb=0 --add_uuid=0  > dkt_train.txt &
![image](https://user-images.githubusercontent.com/102578998/191512876-5a86b967-4020-4c19-a51e-657ae71cab20.png)

**Comment by sonyawong** [comment]
hi, sorry for the mistake that we merged the old version of assist2017_preprocess.py into the main branch. Now the issue is fixed. please try again, thank you.

**Reply by wulongxiaomeem** [reply]
那我需要重新安装吗或者覆盖一下.py文件

**Comment by sonyawong** [comment, marked as answer]
您pull一下main分支，重新运行assist2017的数据预处理即可


| |
Shyann
|
|
***@***.***
|

Signature is customized by Netease Mail Master

On 09/23/2022 21:05, wulongxiaomeem wrote:

那我需要重新安装吗或者覆盖一下.py文件
